components/next_steps_view/next_steps_view.tsx

```diff
@@ -45,12 +67,106 @@ export default class NextStepsView extends React.PureComponent<Props> {
         }
     }
 
-    render() {
-        return (
-            <section
-                id='app-content'
-                className='app__content NextStepsView'
+    getStartingStep = () => {
+        for (let i = 0; i < Steps.length; i++) {
+            if (!this.props.preferences.some((pref) => pref.name === Steps[i].id && pref.value)) {
+                return Steps[i].id;
+            }
+        }
```

Comment:
nit: this is a little hard to parse so it might be worth putting `preferences.some((pref) => pref.name === Steps[i].id && pref.value)` into it's own one-line function that has a clear name